app/src/main/java/com/kanshu/kanshu/LoginActivity.java

```diff
@@ -1,12 +1,33 @@
 package com.kanshu.kanshu;
 
 import android.os.Bundle;
+import android.util.Log;
 import android.view.Menu;
 import android.view.MenuItem;
+import android.view.View;
+import android.widget.EditText;
+
+import com.google.gson.JsonObject;
+
+import org.apache.http.HttpRequest;
+import org.apache.http.auth.AuthenticationException;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.auth.BasicScheme;
+
+import retrofit.Callback;
+import retrofit.RestAdapter;
+import retrofit.RetrofitError;
+import retrofit.client.Response;
+import retrofit.http.Header;
 
 
 public class LoginActivity extends BaseActivity {
 
+    RestAdapter restAdapter = new RestAdapter.Builder().setEndpoint("https://kanshu-ds.herokuapp.com").setLogLevel(RestAdapter.LogLevel.FULL).build();
+
+    KanshuApi kanshuApi = restAdapter.create(KanshuApi.class);
```

Comment:
As recommended by the Retrofit library, can you extract this to make it a singleton used across the whole app? It shouldn't be created every time it is used.